src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSourceHelper.Linux.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers;
+using System.IO;
+
+
+namespace System.Diagnostics.Tracing
+{
+	internal sealed class RuntimeEventSourceHelper
+	{
+		internal static long GetProcessTimes()
```

Comment:
nit: The formatting of this function seems odd.

Reply:
Yes, it looks like spaces have been intermixed with tabs.  All tabs should be converted to spaces.